aadl.library/AADL_Project.sysml

```diff
@@ -1,4 +1,42 @@
+/* SysML v2 Domain Library for AADL:
+ * references: 
+ * SAE AS-5506D AADL appendix A8
+ * OMG SysML v2 release 2024-02 */ 
+
 standard library package AADL_Project {
+	
+	enum def Supported_Dispatch_Protocols {
+		/* Periodic, Sporadic, Aperiodic, Timed, Hybrid, Background, <project-specified> */
+		enum Periodic;
+		enum Sporadic;
+		enum Aperiodic;
+		enum Timed;
+		enum Hybrid;
+		enum Background;
+	}
+	
+	alias Periodic for Supported_Dispatch_Protocols::Periodic;
```

Comment:
We should have a discussion about aliases for enumeration literals. I can see why you added them since enumeration literals in AADL are always referred to by a simple name and are never qualified. However, I feel that this could clutter the namespace. It might be a reasonable approach to use imports in the files that have values.

Reply:
Both approaches are acceptable for me.
It's more a question of modelling guidelines.

The real question is the status of the AADL_Project property set that was never well defined. To which extend is it actually customizable at an end-user or tool level ? Shouldn't we consolidate at least a common subset of these enumeration literals in the standard library. I believe we have enough feedback now to achieve that.